EthOn.rdf

```diff
@@ -1235,21 +1272,7 @@
         <ns:term_status>unstable</ns:term_status>
     </owl:DatatypeProperty>
 
-
-
-    <!-- http://ethon.consensys.net/msgGasPrice -->
-
-    <owl:DatatypeProperty rdf:about="http://ethon.consensys.net/msgGasPrice">
```

Comment:
Hm, there is a new `msgGasCost` but it's not in dev branch yet? We should merge master into dev. @webthethird